release-please-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "release-type": "dart",
+  "packages": {
+    "packages/microsoft_kiota_abstractions": {},
```

Comment:
not sure, maybe the dart release type already handles that, do we need to set the changelog path?

Reply:
Nope it should be defaulting to `CHANGELOG.md` in the package

Reply:
I checked, the default is real actually `CHANGELOG.md`. Should I add explicit paths regardless?

Reply:
no I think we're good with the defaults in that case. Thanks for checking!